aws/cloudformation/components/codeprojects.yml.erb

```diff
@@ -0,0 +1,73 @@
+<%# Note this file must be indented one level, as it is a child of 'Resources'. %>
+  # -----------------------------------------------
+  # Begin codeprojects.org resources
+  # 
+  # This template component includes resources for the codeprojects.org
+  # infrastructure. Currently this is mostly manually deployed.
+  # 
+  # -----------------------------------------------
+
+<%-
+  domain = "codeprojects.org"
```

Comment:
Instead of hard coding the base domain name, can we define a `codeprojects_hostname` attribute in the `CDO` object?

https://github.com/code-dot-org/code-dot-org/blob/7fb913e577d0175dfb94a2bb35b6ec90d1db93b8/lib/cdo.rb#L58-L88

Or add a `CodeProjectsBaseDomainName` Parameter to the template?

Reply:
I've committed some changes that take these into consideration. I need to test out the erb generation before calling it good.